Orange/preprocess/preprocess.py

```diff
@@ -422,3 +504,9 @@ def __call__(self, data):
             data = pp(data)
         return data
 
+    def __repr__(self):
+        repstr = "PreprocessorList(["
+        for preproc in self.preprocessors:
+            repstr +=  repr(preproc) + ", "
+        repstr += "])"
+        return repstr
```

Comment:
Since 
```py
repr(self.preprocessors) == ', '.join(repr(i) for i in self.preprocessors)
```
this could also work with [`Reprable`](https://github.com/biolab/orange3/pull/1507/files#r78161092), right?